### DISCUSSION TITLE
Any thoughts or ideas on migration?

### DISCUSSION THREAD

**Discussion by timreuscher** [007revad/Synology_HDD_db#89] [Q&A] [answered]
Sadly my DS918+ got hit by lightning on Sunday. I have a new DS923+ coming tomorrow to replace the dead unit. I believe my drives are ok (they've spun up when powered), but I had an NVMe drive unofficially working on 7.2. Any ideas on how well that drive is going to migrate? I might be in a position to start from scratch if I have to, as I did replicate the NVMe shares onto other drives. But obviously the easier the better.

Thanks for any thoughts, and much appreciation for the script that enabled the 7.2 upgrade.

**Comment by 007revad** [comment, marked as answer]
1. With the DS923+ turned off, install just the HDDs to start with then turn it on and let it do the drive migration. 
2. Then shut it down and install the NVMe drive(s). 
3. When you boot it back up storage manager should offer to do an "online assemble" of the NVMe volume.
4. You should then syno_hdd_db.sh to get rid of any warning messages.

**Reply by timreuscher** [reply]
Thanks! I've only ever seen the phrase "online assemble" before creating a volume. That does mean I'm losing the data on the NVMe, right?

**Comment by 007revad** [comment]
If you migrate drives from one Synology to another Synology it will ask you to do an "online assemble". You can also  do an "online assemble", instead of rebooting, if the storage pool has crashed or is missing.

Normally you won't lose your volume or data... but there is always a risk.

**Reply by timreuscher** [reply]
OK thanks! I'd just never heard that phrase outside of the volume creation related to NVMes. Appreciate all your feedback! Fingers crossed the drives are still operational and this discussion wasn't just theoretical.

**Comment by timreuscher** [comment]
Went through all of this, getting all data on all drives back! Phew!

For the record, I didn't need to do step 4 as I didn't have any error messages even after a couple of reboots. Not to say I won't need it after an update, but I think it's good for now.
